project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
It does appear in my ".m2/repository/puppetlabs/trapperkeeper-webserver-jetty9/1.5.2-SNAPSHOT/trapperkeeper-webserver-jetty9-1.5.2-SNAPSHOT.pom":

~~~~
...
<dependencies>
  ...
  <dependency>
      <groupId>org.clojure</groupId>
      <artifactId>java.jmx</artifactId>
      <version>0.2.0</version>
      <scope>test</scope>
  </dependency>
  ...
</dependencies>
~~~~

I'll spend a few more minutes seeing if I can figure out something...